src/Database/Factories/HasFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Winter\Storm\Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory as BaseFactory;
+
+trait HasFactory
+{
+    /**
+     * Get a new factory instance for the model.
+     */
+    public static function factory(callable|array|int|null $count = null, callable|array $state = []): BaseFactory
```

Comment:
Should this be returning a Winter BaseFactory, not a Laravel one?

Reply:
Yes, if we also change this in Winter repo's `create:factory` stub.

Reply:
here: `modules/system/console/scaffold/factory/factory.stub`